mlrun/feature_store/retrieval/job.py

```diff
@@ -115,7 +125,7 @@ def target_uri(self):
 
 _default_merger_handler = """
 import mlrun
-from mlrun.feature_store.retrieval import LocalFeatureMerger
+import mlrun.feature_store.retrieval
 from mlrun.datastore.targets import get_target_driver
 def merge_handler(context, vector_uri, target, entity_rows=None, 
                   timestamp_column=None, drop_columns=None, with_indexes=None, query=None):
```

Comment:
```suggestion
                  timestamp_column=None, drop_columns=None, with_indexes=None, query=None, engine_args=None):
```
Per my suggestion above - simply pass them in the task.